docs/content/guides/accessibility/accessibility.md

```diff
@@ -0,0 +1,185 @@
+### Shortcut Keys Configuration
+Handsontable provides an extensive amount of [built-in shortcut keys](https://handsontable.com/docs/javascript-data-grid/keyboard-shortcuts/) and the ability to customize them, but some of them stand out. <br><br>
+For instance, one great example would be opening the menus. The user gets access to variety of actions. With navigable headres enabled it gets easier for disabled users to navigate across the grid with simple shortcut keys.
+| Action/Setting                   | Data grid mode (Navigable headers)                                           | Spreadsheet mode (Non-navigable headers)                                                    |
+|----------------------------------|------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
+| Open column menu                 | macOS: <kbd>Cmd</kbd> + <kbd>Enter</kbd><br>Windows: <kbd>Ctrl</kbd> + <kbd>Enter</kbd>                | macOS: <kbd>Shift</kbd> + <kbd>Option</kbd> + <kbd>i</kbd><br>Windows: <kbd>Shift</kbd> + <kbd>Alt</kbd> + <kbd>i</kbd> |
+| Sort data                        | <kbd>Shift</kbd> + <kbd>Enter</kbd>                                         | Use of a custom context menu item is required                                               |
+| Clear filters                    | macOS: <kbd>Alt</kbd> + <kbd>A</kbd><br>Windows: <kbd>Option</kbd> + <kbd>A</kbd>                    | macOS: <kbd>Alt</kbd> + <kbd>A</kbd><br>Windows: <kbd>Option</kbd> + <kbd>A</kbd>           |
+| Select the entire column         | <kbd>Ctrl</kbd> + <kbd>space</kbd>                                          | <kbd>Ctrl</kbd> + <kbd>space</kbd>                                                           |
+| Select the entire row            | <kbd>Shift</kbd> + <kbd>space</kbd>                                         | <kbd>Shift</kbd> + <kbd>space</kbd>                                                          |
+| Invoke context menu              | macOS: <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>\\</kbd> <br> <kbd>Shift</kbd> + <kbd>F10</kbd> <br> <br> Windows: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>\\</kbd> <br> <kbd>Shift</kbd> + <kbd>F10</kbd> | macOS: <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>\\</kbd> <br>Windows: <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>\\</kbd> <br> <kbd>Shift</kbd> + <kbd>F10</kbd> |
+
+
+This is the exerpt from [this]((https://handsontable.com/docs/javascript-data-grid/keyboard-shortcuts/)) page. 
+
+## Screen Readers
+Recognized as a composite widget, Handsontable adopts the ARIA role of treegrid, ensuring that it adheres to established standards. To enhance the experience for visually impaired users, Handsontable employs ARIA attributes, which provide screen readers with a clearer understanding of the data's structure and content.
```

Comment:
Think we should change this to something along the lines of "Recognized by WAI-ARIA as a composite widget, Handsontable adopts"

Reply:
changed